### PR TITLE
Score UI feature

### PR DESCRIPTION
**Pull request by jorlol**

### CODE REVIEW

File: Assets/Scripts/FallManager.cs [hunk 1]
```diff
@@ -32,7 +32,8 @@ private void Update()
         }
     }
 
-    private IEnumerator EnableTarget(bool active) {
+    private IEnumerator EnableTarget(bool active)
+    {
         yield return new WaitForSeconds(WaitForSeconds);
         FindObjectOfType <BuildingTracker> ().ShowSummary ();
```
Comment (Lansenou) [inline Assets/Scripts/FallManager.cs]: Perhaps check if the object is found, or set the buildingtracker in the inspector.

File: Assets/Scripts/PopupController.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+{
+
+    private static FloatingPopupText popupText;
+    private static GameObject canvas;
+
+    public static void Initialize()
+    {
+        if (!canvas)
+        {
+            canvas = GameObject.Find("Canvas");
```
Comment (Lansenou) [inline Assets/Scripts/PopupController.cs]: Let's not use [GameObject.Find()](https://akbiggs.silvrback.com/please-stop-using-gameobject-find).

File: Assets/Scripts/PopupController.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        }
+        if (!popupText) { 
+            popupText = Resources.Load<FloatingPopupText>("Prefabs/PopupTextParent");
+        }
+    }
+
+public static void CreateFloatingText(string text, Color color)
```
Comment (Lansenou) [inline Assets/Scripts/PopupController.cs]: Indentation seems to be wrong.

File: Assets/Scripts/FloatingPopupText.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+using UnityEngine.UI;
+
+public class FloatingPopupText : MonoBehaviour {
+
+    public Animator animator;
+    // Use this for initialization
+    void Start ()
+    {
+        AnimatorClipInfo[] clipInfo = animator.GetCurrentAnimatorClipInfo(0);
+        Destroy(gameObject, clipInfo[0].clip.length);
```
Comment (Lansenou) [inline Assets/Scripts/FloatingPopupText.cs]: Not important but you can call a function from an animation, that might be a neater way to destroy to object.
https://docs.unity3d.com/Manual/animeditor-AnimationEvents.html

File: Assets/Scripts/HighScore.cs [hunk 5]
```diff
@@ -16,14 +16,14 @@ private void Start()
     {
         highScore = PlayerPrefs.GetFloat("HighScore", 0);
         text2.text = "HighScore: " + highScore.ToString("0");
-        text.text = "Score:" + (CurrentScore = 0);
+        text.text = "" + (CurrentScore = 0);
```
Comment (Lansenou) [inline Assets/Scripts/HighScore.cs]: Can you please revert this change?

File: Assets/Scripts/HighScore.cs [hunk 6]
```diff
@@ -16,14 +16,14 @@ private void Start()
...
     }
 
     private void Update()
     {
         if (CurrentScore > prevScore)
         {
-            text.text = "Score:" + (prevScore = CurrentScore).ToString("0");
+            text.text = "" + (prevScore = CurrentScore).ToString("0");
```
Comment (Lansenou) [inline Assets/Scripts/HighScore.cs]: Revert it here aswell.

File: Assets/Scripts/PopupController.cs (hunk 2 above)
Comment (jorlol) [inline Assets/Scripts/PopupController.cs]: ohja deze is wel echt van mij lol

Comment (Lansenou) [inline Assets/Scripts/PopupController.cs]: What about this?
http://answers.unity3d.com/answers/644752/view.html

Comment (Lansenou) [inline Assets/Scripts/PopupController.cs]: Except then use FindObjectOfType<Canvas>()
